### PR TITLE
feat(global): DOMA-5542 bridge modals

### PR DESCRIPTION
**Pull request by SavelevMatthew**
https://user-images.githubusercontent.com/50069872/225277910-dd7177b3-2839-4a12-9be7-b83e2bb9c076.mov

### CODE REVIEW

File: packages/ui/src/components/Modal/style.less [hunk 1]
```diff
@@ -78,6 +78,7 @@
 
       .condo-modal-wrapper {
         width: fit-content;
+        min-width: 100%;
```
Comment (dkoviazin) [inline packages/ui/src/components/Modal/style.less:81]: maybe, width?

File: packages/ui/src/components/Modal/useModal.tsx [hunk 2]
```diff
@@ -0,0 +1,141 @@
+import React, { useCallback, useEffect, useImperativeHandle, useRef, useState } from 'react'
+
+import { Modal } from './modal'
+
+import { usePatchElement } from '../_utils/hooks/usePatchElement'
+
+import type { ModalProps } from './modal'
+
+type AnyArgs = Array<any>
```
Comment (dkoviazin) [inline packages/ui/src/components/Modal/useModal.tsx:9]: do we really have no idea what args will be?

File: packages/ui/src/components/Modal/useModal.tsx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+type ModalConfig = Omit<ModalProps, 'open'> & {
+    onCancel?: (...args: AnyArgs) => any
+}
+
+let uuid = 0
```
Comment (dkoviazin) [inline packages/ui/src/components/Modal/useModal.tsx:15]: UUID (Universal Unique Identifier) is a 128-bit value

**Review by dkoviazin — APPROVED** [review]

File: packages/ui/src/components/Modal/style.less (hunk 1 above)
Comment (SavelevMatthew) [reply packages/ui/src/components/Modal/style.less:81]: nah :)

File: packages/ui/src/components/Modal/useModal.tsx (hunk 2 above)
Comment (SavelevMatthew) [reply packages/ui/src/components/Modal/useModal.tsx:9]: Antd does not, that's part of their original code

Comment (SavelevMatthew) [reply packages/ui/src/components/Modal/useModal.tsx:9]: and I guess they're taking logic from `rc-dialog`, so it's nested type def, that's why it's any

File: packages/ui/src/components/Modal/modal.tsx [hunk 4]
```diff
@@ -68,7 +68,8 @@ const Modal: React.FC<ModalProps> = (props) => {
             centered
             closable
             className={classes}
-            closeIcon={<Cross />}
+            // closeIcon={<Close size='small' color={colors.black}/>}
```
Comment (Alllex202) [inline packages/ui/src/components/Modal/modal.tsx:71]: maybe remove it?

**Review by abshnko — APPROVED** [review]

**Review by Alllex202 — APPROVED** [review]

**Pull request merged**